tests/bounded_bls12381.rs

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//! Test suite for the Web and headless browsers.
+
+#![cfg(target_arch = "wasm32")]
+extern crate wasm_bindgen_test;
+
+use arrayref::array_ref;
+use bbs::{prelude::*, signature::BlindSignature};
+use std::convert::TryInto;
+use wasm::prelude::*;
+use wasm::U8_STRING;
```

Comment:
```suggestion
use wasm::{prelude::*, BbsVerifyResponse, U8_STRING};
```
Add `BbsVerifyResponse` for the test below